lib/mix/tasks/absinthe.schema.json.ex

```diff
@@ -80,6 +80,12 @@ defmodule Mix.Tasks.Absinthe.Schema.Json do
 
     opts = parse_options(argv)
 
+    # Start for Absinthe.Schema.PersistentTerm
```

Comment:
Would this also need to be added to `absinthe.schema.sdl`?

Reply:
Great question! I'm unsure, we had no problem running `absinthe.schema.sdl` locally.  It was only when we tried `absinthe.schema.json`.

Reply:
Could you please also add the same change to `absinthe.schema.sql`? This could fix #1054